pkg/services/servicebus/queue_bind.go

```diff
@@ -22,6 +22,7 @@ func (qm *queueManager) GetCredentials(
 	return queueCredentials{
 		ConnectionString: string(pdt.ConnectionString),
 		PrimaryKey:       string(pdt.PrimaryKey),
+		QueueName:        string(dt.ServiceBusQueueName),
 		QueueURL: fmt.Sprintf(
 			"https://%s.servicebus.windows.net/%s",
```

Comment:
The sovereign clouds have different suffixes for the endpoint: https://github.com/Azure/go-autorest/blob/master/autorest/azure/environments.go#L102.

Reply:
If so, I would rather remove this field...

Reply:
I am okay removing it. Or you can modify the connection string to get the correct url. It's up to you.